src/components/rating/rating.jsx

```diff
@@ -0,0 +1,33 @@
+'use strict';
+var React = require('react');
+
+module.exports = React.createClass({
+  propTypes: {
+    children: React.PropTypes.any,
+    rating: React.PropTypes.number.isRequired,
+    outOf: React.PropTypes.number,
+    blankIcon: React.PropTypes.node
+  },
+
+  render: function() {
+    var $rating = [];
```

Comment:
I am not sure about this convention either, I think it's a good convention for cached jquery selectors but I'm not sure this is it. I've not changed this, though unfortunately it looks like I have - the original code is here https://github.com/holidayextras/ui-toolkit/blob/master/src/components/rating/code/templates/ratingTemplate.jsx